missing symbols defined in includes/rts/Bytecodes.h when cross-compiling

according to steps in `docs/cross-compile.md`

```
user error (Development.Shake.cmd, system command failed
Command: /usr/local/bin/ghc -Wall -hisuf hi -osuf o -hcsuf hc -static -hide-all-packages -no-user-package-db '-package-db _build/stage0/lib/package.conf.d' '-this-unit-id ghc-8.5' '-package-id array-0.5.2.0' '-package-id base-4.10.1.0' '-package-id binary-0.8.5.1' '-package-id bytestring-0.10.8.2' '-package-id containers-0.5.10.2' '-package-id deepseq-1.4.3.0' '-package-id directory-1.3.0.2' '-package-id filepath-1.4.1.2' '-package-id ghc-boot-8.5' '-package-id ghc-boot-th-8.5' '-package-id ghci-8.5' '-package-id hpc-0.6.0.3' '-package-id process-1.6.1.0' '-package-id template-haskell-2.14.0.0' '-package-id time-1.8.0.2' '-package-id transformers-0.5.5.0' '-package-id unix-2.7.2.2' -i -i_build/stage0/compiler/build -i_build/stage0/compiler/build/autogen -icompiler/backpack -icompiler/basicTypes -icompiler/cmm -icompiler/codeGen -icompiler/coreSyn -icompiler/deSugar -icompiler/ghci -icompiler/hsSyn -icompiler/iface -icompiler/llvmGen -icompiler/main -icompiler/nativeGen -icompiler/parser -icompiler/prelude -icompiler/profiling -icompiler/rename -icompiler/simplCore -icompiler/simplStg -icompiler/specialise -icompiler/stgSyn -icompiler/stranal -icompiler/typecheck -icompiler/types -icompiler/utils -icompiler/vectorise -I_build/generated -I_build/stage0/compiler/build -I_build/stage0/compiler/build/. -I_build/stage0/compiler/build/parser -I_build/stage0/compiler/build/utils -Icompiler/. -Icompiler/parser -Icompiler/utils -I/usr/local/haskell/ghc-8.2.2-x86_64/lib/ghc-8.2.2/process-1.6.1.0/include -I/usr/local/haskell/ghc-8.2.2-x86_64/lib/ghc-8.2.2/directory-1.3.0.2/include -I/usr/local/haskell/ghc-8.2.2-x86_64/lib/ghc-8.2.2/unix-2.7.2.2/include -I/usr/local/haskell/ghc-8.2.2-x86_64/lib/ghc-8.2.2/time-1.8.0.2/include -I/usr/local/haskell/ghc-8.2.2-x86_64/lib/ghc-8.2.2/bytestring-0.10.8.2/include -I/usr/local/haskell/ghc-8.2.2-x86_64/lib/ghc-8.2.2/base-4.10.1.0/include -I/usr/local/haskell/ghc-8.2.2-x86_64/lib/ghc-8.2.2/integer-gmp-1.0.1.0/include -I/usr/local/haskell/ghc-8.2.2-x86_64/lib/ghc-8.2.2/include -I_build/generated -optc-I_build/generated -optP-include -optP_build/stage0/compiler/build/autogen/cabal_macros.h -optc-fno-stack-protector -odir _build/stage0/compiler/build -hidir _build/stage0/compiler/build -stubdir _build/stage0/compiler/build -optc-Werror=unused-but-set-variable -optc-Wno-error=inline -c compiler/ghci/ByteCodeAsm.hs -o _build/stage0/compiler/build/ByteCodeAsm.o -O0 -H64m -Wall -Wno-name-shadowing -Wnoncanonical-monad-instances -Wnoncanonical-monadfail-instances -Wnoncanonical-monoid-instances -this-unit-id ghc -XHaskell2010 -XNoImplicitPrelude -DSTAGE=1 -O -fno-warn-deprecated-flags
Exit code: 1
Stderr:
compiler/ghci/ByteCodeAsm.hs:354:36: error:
    Variable not in scope: bci_PUSH8 :: Word16
    |
354 |   PUSH8 o1                 -> emit bci_PUSH8 [SmallOp o1]
    |                                    ^^^^^^^^^

compiler/ghci/ByteCodeAsm.hs:355:36: error:
    Variable not in scope: bci_PUSH16 :: Word16
    |
355 |   PUSH16 o1                -> emit bci_PUSH16 [SmallOp o1]
    |                                    ^^^^^^^^^^

compiler/ghci/ByteCodeAsm.hs:356:36: error:
    Variable not in scope: bci_PUSH32 :: Word16
    |
356 |   PUSH32 o1                -> emit bci_PUSH32 [SmallOp o1]
    |                                    ^^^^^^^^^^

compiler/ghci/ByteCodeAsm.hs:357:36: error:
    Variable not in scope: bci_PUSH8_W :: Word16
    |
357 |   PUSH8_W o1               -> emit bci_PUSH8_W [SmallOp o1]
    |                                    ^^^^^^^^^^^

compiler/ghci/ByteCodeAsm.hs:358:36: error:
    Variable not in scope: bci_PUSH16_W :: Word16
    |
358 |   PUSH16_W o1              -> emit bci_PUSH16_W [SmallOp o1]
    |                                    ^^^^^^^^^^^^

compiler/ghci/ByteCodeAsm.hs:359:36: error:
    Variable not in scope: bci_PUSH32_W :: Word16
    |
359 |   PUSH32_W o1              -> emit bci_PUSH32_W [SmallOp o1]
    |                                    ^^^^^^^^^^^^

compiler/ghci/ByteCodeAsm.hs:374:36: error:
    Variable not in scope: bci_PUSH_PAD8 :: Word16
    |
374 |   PUSH_PAD8                -> emit bci_PUSH_PAD8 []
    |                                    ^^^^^^^^^^^^^

compiler/ghci/ByteCodeAsm.hs:375:36: error:
    • Variable not in scope: bci_PUSH_PAD16 :: Word16
    • Perhaps you meant data constructor ‘PUSH_PAD16’ (imported from ByteCodeInstr)
    |
375 |   PUSH_PAD16               -> emit bci_PUSH_PAD16 []
    |                                    ^^^^^^^^^^^^^^

compiler/ghci/ByteCodeAsm.hs:376:36: error:
    • Variable not in scope: bci_PUSH_PAD32 :: Word16
    • Perhaps you meant data constructor ‘PUSH_PAD32’ (imported from ByteCodeInstr)
    |
376 |   PUSH_PAD32               -> emit bci_PUSH_PAD32 []
    |                                    ^^^^^^^^^^^^^^

compiler/ghci/ByteCodeAsm.hs:378:39: error:
    Variable not in scope: bci_PUSH_UBX8 :: Word16
    |
378 |                                  emit bci_PUSH_UBX8 [Op np]
    |                                       ^^^^^^^^^^^^^

compiler/ghci/ByteCodeAsm.hs:380:39: error:
    • Variable not in scope: bci_PUSH_UBX16 :: Word16
    • Perhaps you meant data constructor ‘PUSH_UBX16’ (imported from ByteCodeInstr)
    |
380 |                                  emit bci_PUSH_UBX16 [Op np]
    |                                       ^^^^^^^^^^^^^^

compiler/ghci/ByteCodeAsm.hs:382:39: error:
    • Variable not in scope: bci_PUSH_UBX32 :: Word16
    • Perhaps you meant data constructor ‘PUSH_UBX32’ (imported from ByteCodeInstr)
    |
382 |                                  emit bci_PUSH_UBX32 [Op np]
    |                                       ^^^^^^^^^^^^^^
)
```

--- a/src/Settings/Builders/Common.hs
+++ b/src/Settings/Builders/Common.hs
@@ -29,9 +29,7 @@ cIncludeArgs = do
     gmpIncludeDir   <- getSetting GmpIncludeDir
     ffiIncludeDir   <- getSetting FfiIncludeDir
 
-    cross   <- expr crossCompiling
-    compilerOrGhc <- package compiler ||^ package ghc
-    mconcat [ not (cross && compilerOrGhc) ? arg "-Iincludes"
+    mconcat [ arg "-Iincludes"
             , arg $ "-I" ++ root -/- generatedDir
             , arg $ "-I" ++ path
             , pure . map ("-I"++) . filter (/= "") $ [iconvIncludeDir, gmpIncludeDir]

